sdk/keyvault/keyvault-keys/src/core/keyVaultClientContext.ts

```diff
@@ -9,9 +9,7 @@
  */
 
 import * as coreHttp from "@azure/core-http";
-
-const packageName = "@azure/keyvault-keys";
-const packageVersion = "0.0.1";
+import { USER_AGENT } from "./utils/constants";
```

Comment:
This is definitely going to get overwritten by code generation, so I'd recommend doing the user agent calculation outside of the generated code entirely.